packages/node_modules/@webex/plugin-meetings/src/meetings/index.js

```diff
@@ -253,14 +253,8 @@ export default class Meetings extends WebexPlugin {
         this.create(data.locus, _LOCUS_ID_).then((newMeeting) => {
           meeting = newMeeting;
 
-          if (data.eventType === LOCUSEVENT.DIFFERENCE) {
-            // its a delta object and we have a new meeting
-            meeting.locusInfo.initialSetup(data.locus, meeting);
-          }
-          else {
-            // Its a new meeting and have a fresh locus object
-            meeting.locusInfo.initialSetup(data.locus);
-          }
+          // Its a new meeting so initialize with locus data
+          meeting.locusInfo.initialSetup(data.locus);
```

Comment:
After this change the `initialSetup` method in locus-info is never called with a second "meeting" argument. Should the code that handles this be removed from locusInfo?

Reply:
We will need to pass the meeting object i guess as the logic for delta event is different then initialSetup 

Can you test a meeting with more then 10 or 15 participant , We will see issues with members list there as the delta is like appending and initial-setup is fully replacing it 

You can may be set the first event to initialSetup and reset of the event to delta and that should work

Reply:
@Coread Good point. Will simplify intialSetup().

@arun3528 But initialSetup() should only be called in the case where we do not find a matching meeting object, right?  So only the first time, then onDeltaEvent() for the rest.

Reply:
we are assuming that the first event will always be a full event

Reply:
The first event should be a full event if syncMeetings() does its job (which is another issue).  Even if something goes wrong and syncMeetings does not return a full locus for a meeting, then when the first locus difference event arrives, handleLocusEvent() will create a meeting object with whatever locus info is there (the same as it did before).  The only difference will be that initialSetup() is called with the partial locus info.  I think that's fine, and at the very least, better than what happened before.

I was able to get 12 aMTbot participants to join a scheduled space meeting (according to aMTbot), though I only every saw evidence of 6 "waiting to join" in the web client. I did see mercury events for the participants and getAllMeetings()[0].members.membersCollection.members shows them all.  Not sure if that is enough to address your request.